lib/globalize/active_record/class_methods.rb

```diff
@@ -56,7 +56,7 @@ def translation_class
             klass = self.const_set(:Translation, Class.new(Globalize::ActiveRecord::Translation))
           end
 
-          klass.belongs_to :globalized_model, :class_name => self.name, :foreign_key => translation_options[:foreign_key], inverse_of: :translations
+          klass.belongs_to :globalized_model, :class_name => self.name, :foreign_key => translation_options[:foreign_key], inverse_of: :translations, touch: translation_options[:touch] || false
```

Comment:
here we could use `fetch`:

```ruby
klass.belongs_to :globalized_model, 
  class_name: self.name, 
  foreign_key: translation_options[:foreign_key], 
  inverse_of: :translations, 
  touch: translation_options.fetch(:touch, false)
```

I find this makes it easier to read and understand that the `|| false` applies to the `touch` option.